### PR TITLE
Changed font to Avenir Next

### LINKED ISSUE #20: Change font to Avenir Next
Change font to Avenir Next, with "regular" as the main weight for content while "bold" used for buttons. (An alternative is Lato if we can't find a free one (https://www.google.com/fonts/specimen/Lato)

### PR DESCRIPTION
**Pull request by rahulbisht12**
Fixes #20

### CODE REVIEW

**Comment by AkankshaBodhankar** [comment]
Can you post a screenshot here ?

File: css files/gethelpnow-style.css [hunk 1]
```diff
@@ -61,7 +75,7 @@ body {
     color: #ffffff;
     float: right;
     font-size: 28px;
-    font-weight: bold;
+	font-family: 'Avenir Next Bold';
```
Comment (AkankshaBodhankar) [inline css files/gethelpnow-style.css]: remove extra spacing

**Comment by AkankshaBodhankar** [comment]
code looks good! Just adjust the spacing and resolve the conflicts. I would suggest it's better to make a new PR with the conflicts resolved and changes made. Once you do this you will be done with the task

**Comment by rahulbisht12** [comment]
@AkankshaBodhankar I have updated the PR and squashed all the commits so its now easy to merge the changes. Here are the screenshots. :)

p2
p3

**Comment by rahulbisht12** [comment]
@AkankshaBodhankar Why the "Travis CI build failed" for this commit ?

**Comment by AkankshaBodhankar** [comment]
That's an issue from our end don't worry about it. You work looks good. Keep it up !

**Comment by rahulbisht12** [comment]
@AkankshaBodhankar When this PR will be merged ?

**Comment by AkankshaBodhankar** [comment]
Right now we are still reviewing the PR's so, merging won't be done immediately

**Comment by dannymout** [comment]
Question: Do you have the license for using Avenir Next in an app? Otherwise, it could lead to legal problems with whoever owns the rights to it. I highly suggest going with a free font, [Google Fonts](fonts.google.com) has a selection of high-quality fonts that are free. Also, an interesting approach I like to use is writing the CSS to use the main system font, ex. (macOS/iOS: San Francisco, Android/ChromeOS: Roboto, Windows: Segoe UI)

This code allows you to do so: `font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
`

**Comment by AkankshaBodhankar** [comment]
@dannymout Good suggestion! We can use free fonts instead. PR has already been made for this issue. Once it is merged feel free to make the changes.

**Pull request merged**